### PR TITLE
added 1 in boomer

### PR DESCRIPTION
**Pull request by rishavr920**
i added number 1 in boomer merge it plzz

### CODE REVIEW

**Comment by rishavr920** [comment]
#560

File: Fonts/Boomer/boomer.h [hunk 1]
```diff
@@ -671,18 +671,22 @@ class Boomer : public Fonts
...
 
-    /*
-        char **one()
+    char **one()
         {
-            char **character = getCharGrid(ROWS,COLS);
+            char **character = getCharGrid(6,7);
 
             //Enter the character grid in ROWS X COLS
+            character[0][0]=character[1][0]=character[2][0]=character[3][0]=character[4][0]=  character[5][0]=  character[6][0]=  character[7][0]=' ';
```
Comment (AnshumanMahato) [inline Fonts/Boomer/boomer.h:679]: image

error upon compilation... please resolve

You have taken a grid of 6 rows and 7 columns but you using 8 rows from what I can see. Please test your code before pushing changes.

**Review by AnshumanMahato — CHANGES_REQUESTED** [review]

File: Fonts/Boomer/boomer.h (hunk 1 above)
Comment (rishavr920) [reply Fonts/Boomer/boomer.h:691]: sir I have changed the grid size please check if it alright merge it.

**Comment by rishavr920** [comment]
sir I have changed the grid size please check if it alright merge it.

**Comment by AnshumanMahato** [comment]
hey... did you test your code? It is giving this
image

What's expected is this 
image

If you are getting stuck somewhere, feel free to ask for help 😊

**Pull request closed without merging**